test/VueSimpleIcon.test.ts

```diff
@@ -9,17 +9,7 @@ describe('VueSimpleIcon', () => {
       }
     });
 
-    expect(wrapper.html()).toMatchSnapshot();
-  });
-
-  it('should render icon with multiple paths', () => {
-    const wrapper = shallowMount(VueSimpleIcon, {
-      propsData: {
-        name: 'Simple Icons'
-      }
-    });

```

Comment:
Why removing this?

Reply:
From a0f314b79ec882effb116b1bc0feb84b206a449e:

> remove explicit test for multiple paths since no icons have multiple paths since simple-icons/simple-icons#1275